tests/psa-client-server/psasim/test/run_test.sh

```diff
@@ -30,8 +30,8 @@ function wait_for_server_startup() {
 
 clean_run
 
-./psa_partition -k > psa_partition.log 2>&1 &
+./psa_partition -k &
```

Comment:
> The PSA sim logs aren't very many lines, so show them during tests

I think this will change in the future as soon as we add more tests. What about a `tee` here? So that we can have both the log file and output on the console?
Or we can make log file optional using input parameters to the `run_test.sh` script?

Reply:
As you say, we are likely going to be iterating on this component in the future. Since right now it's useful to have the output on the console, and everything gets cleaned up at the end, I'd rather keep this as it is here now and change in the future if we want to